internal/packmgr/installer.go

```
@@ -0,0 +1,406 @@
+			return InstalledRecord{}, fmt.Errorf("包 %s@%s 已安装", source.Manifest.ID, source.Manifest.Version)
+		}
+		if err := validateDependents(ctx, root, source.Manifest.ID, source.Manifest.Version); err != nil {
+			return InstalledRecord{}, err
+		}
+	}
+	// 原子发布：先在安装根内创建临时阶段目录，再 rename 为目标。
+	stageDir, err := os.MkdirTemp(root, stagePrefix)
+	if err != nil {
+		return InstalledRecord{}, err
+	}
+	defer os.RemoveAll(stageDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)